packages/template-retail-react-app/app/components/product-view-modal/bundle.jsx

```diff
@@ -71,7 +78,7 @@ const BundleProductViewModal = ({product: bundle, isOpen, onClose, updateCart, .
                             <Box marginBottom={6}>
                                 <ProductView
                                     showFullLink={false}
-                                    showImageGallery={false}
+                                    showImageGallery={trueIfMobile}
```

Comment:
But if you're on _mobile, then we'll want to display the image gallery that comes included in `ProductView` component.

Reply:
So, question here. Are there two galleries... one for mobile and one for desktop? If yes, is there any way we can make the one gallery responsive instead of introducing a show/hide filter?